Source/Services/Services.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="Grpc" Version="$(GrpcVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="$(MicrosoftExtensionsVersion)" />
     <PackageReference Include="System.Reactive" Version="$(RxVersion)" />
+    <PackageReference Include="Contrib.Grpc.Core.M1" Version="$(GrpcVersion)" />
```

Comment:
I need to test this on my intel Mac.

Reply:
It works, they are put in the separate directories of the build output under `runtimes/<platform>/native` so they don't collide with each other.